corehq/apps/export/static/export/spec/list_exports.spec.mo.js

```diff
@@ -0,0 +1,166 @@
+describe('ListExportsController Unit Tests', function() {
+    var $httpBackend, createController, $rootScope, currentScope;
+
+    var mockBackendUrls = {
+        GET_EXPORTS_LIST: '/fake/exports/list',
+        UPDATE_EMAILED_EXPORT_DATA: '/fake/exports/update/data'
+    };
+
+    beforeEach(function () {
+        var listExportsTestApp = angular.module('ngtest.ListExportsApp', ['hq.list_exports']);
+        listExportsTestApp.config(["djangoRMIProvider", function (djangoRMIProvider) {
+            djangoRMIProvider.configure({
+                get_exports_list: {
+                    url: mockBackendUrls.GET_EXPORTS_LIST,
+                    headers: {
+                        'DjNg-Remote-Method': 'get_exports_list'
+                    },
+                    method: 'auto'
+                },
+                update_emailed_export_data: {
+                    url: mockBackendUrls.UPDATE_EMAILED_EXPORT_DATA,
+                    headers: {
+                        'DjNg-Remote-Method': 'update_emailed_export_data'
+                    },
+                    method: 'auto'
+                }
+            });
+        }]);
+        module('ngtest.ListExportsApp');
+        // Kickstart the injectors previously registered with calls to angular.mock.module
+        inject(function () {});
+    });
+
+    beforeEach(inject(function($injector) {
+        $rootScope = $injector.get('$rootScope');
+        $httpBackend = $injector.get('$httpBackend');
+        var $controller = $injector.get('$controller');
+        createController = function () {
+            currentScope = $rootScope.$new();
+            return $controller('ListExportsController', {'$scope': currentScope});
+        };
+
+    }));
+
+    describe('Initialization', function() {
+
+        afterEach(function() {
+            $httpBackend.verifyNoOutstandingExpectation();
+            $httpBackend.verifyNoOutstandingRequest();
+        });
+
+        it('registers a non-blank list of exports', function() {
+            $httpBackend
```

Comment:
another way to mock out the backend that doesn't require modifying the existing code is to use sinon's fake server functionality: http://sinonjs.org/docs/#fakeServer